assets/js/messages.json

```diff
@@ -326,6 +326,14 @@
                 "fillname": []
             }
         ],
+        [
+            {
+                "project": ["mc"],
+                "name": "Provide seed and coordinates",
+                "message": "*Thank you for your report!*\nHowever, this issue has been temporarily closed as {color:#FF5722}*Awaiting Response*{color}.\n\nTo be able to reproduce this, we need to know some information about the the world you are playing in.\nPlease provide the following information as text, attaching a screenshot showing them is not enough:\n- World Type\n- Seed: Can be obtained by using the {{/seed}} command\n- Coordinates: Can be copied to clipboard by pressing {{F3}} + {{C}}\n\nOnce the requested information has been delivered, the report will be reopened automatically.\n\n%quick_links%",
```

Comment:
Awaiting response messages are usually in the scheme of

```
_We do not have enough information to find the cause of this issue._

<REQUEST>

~This issue has been temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~

%quick_links%
```

Reply:
"Force debug crash report" also starts with "However, this issue...".
Maybe it would make sense to make it consistent for all messages then. Either resolution at the beginning or the end?
Should I change it then in this pull request?

And I did not include
>We do not have enough information to find the cause of this issue.
on purpose because even with seed and coordinates we will (very likely) not have enough information to find the cause, we can at most reproduce it (though Mojang might find the cause).

Reply:
Oh yeah that's different.
Both Attach Crash Report and Attach World File use the one I posted though.

Finding the cause is probably meant as a synonym for reproducing it, at least for the world file. While attaching a world file might yield a cause that makes the report invalid, I think it would be fair to say the we always assume a valid report unless proven otherwise.
But "We do not have enough information to reproduce this issue" might be more fit for this one and Attach World File.

Reply:
So replace
> To be able to reproduce this, we need to know some information about the the world you are playing in.

with this?
> _We do not have enough information to reproduce this issue._

And then end the message with this?
```
~This issue has been temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
```

Reply:
Sounds good to me